src/main/java/team/gif/robot/RobotMap.java

```diff
@@ -11,7 +11,8 @@
     // Other Motors
     public static final int MOTOR_INTAKE = 3;
     public static final int MOTOR_MID_INDEX = 10;
-    public static final int MOTOR_BELT = 2;
+    public static final int MOTOR_BELT_PRACTICE = 31; //2
+    public static final int MOTOR_BELT_COMPBOT = 2;

```

Comment:
leave this as MOTOR_BELT and use:
public static final int MOTOR_BELT = Robot.isCompBot ? 2 : 31;

Reply:
The reason we separated this out was because instantiating two different motor controllers on the same port cause the belt to not run

Reply:
Ah, yes. That makes sense. We can use the isCompBot if they were using the same controller types but not if they are different. I wish Java had something similar to a #define like C/C++ has.

Reply:
Yeah, that would be really helpful. I can think of a couple ways to abstract this or make it cleaner, but none of them seem worth the effort at this point in time